apiserver/worker/worker.py

```diff
@@ -160,6 +160,11 @@ def executeCompileTask(user_id, bot_id, backend):
         try:
             if didCompile:
                 logging.debug("Bot did compile\n")
+
+                # Make things group-readable
+                subprocess.call([
+                    "sudo", "-H", "-u", user, "-s", "chmod", "-R", "770", temp_dir
```

Comment:
Maybe g+r instead?

Reply:
Mostly to align the comment with the command. It's a bit of a cognitive dissonance right now.